Automattic/jetpack
# Improve AMP compatibility for the Mixcloud shortcode

#### Changes proposed in this Pull Request:
* In the `[mixcloud]`shortcode, adds a `sandbox` attribute which allows opening links in a new window and fixes a console error on page load.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* This enhances an existing feature.

#### Testing instructions:
* Ensure the AMP plugin is active
* In AMP Settings, select 'Transitional' mode
* In Jetpack > Settings > Writing, ensure 'Compose using shortcodes' option is toggled on
* Add a shortcode block with: `[mixcloud width=500 height 100 mini="true" hide_followers="true" hide_artwork="true" light="true"]https://www.mixcloud.com/DJOneF/djonef-mashup-mix-june-2020/[/mixcloud]`
* View the page with `?amp` in the query string
* Click a link inside the Twitch TV embed
* Expected: Link opens up in a new window/tab, and does not throw a console error

#### Proposed changelog entry for your changes:
* Improved Mixcloud shortcode AMP compatibility
